cliquet/tests/test_initialization.py

```diff
@@ -59,6 +59,14 @@ def test_set_the_project_name_from_settings_even_if_specified(self):
         self.assertEqual(config.registry.settings['cliquet.project_name'],
                          'kinto')
 
+    def test_set_version_prefix_redirect_to_false(self):
+        config = Configurator(
+            settings={'cliquet.version_prefix_redirect_enabled': False})
+        cliquet.initialize(config, '0.0.1', 'kinto')
+        redirect_enabled = config.registry.settings[
+            'cliquet.version_prefix_redirect_enabled']
+        self.assertFalse(redirect_enabled)
```

Comment:
I was reading this again, and I think that this test can be removed. Indeed it only tests that Pyramid is storing settings in registry, which isn't something in the scope of the pull-request :)